src/hct_mis_api/apps/dashboard/services.py

```diff
@@ -243,9 +241,7 @@ def refresh_data(cls, business_area_slug: str = "global") -> ReturnDict:
             .annotate(
                 country=F("business_area__name"),
                 year=ExtractYear(Coalesce("delivery_date", "entitlement_date", "status_date")),
-                programs=Coalesce(F("household__program__name"), Value("Unknown program")),
```

Comment:
programs  should not be used... correct @johniak ?

Reply:
Why? they want to see the figures by program. at least on the country.